test/Microsoft.Health.Fhir.Tests.E2E/SmartProxy/SmartProxyTests.cs

```diff
@@ -111,6 +111,10 @@ void Advance()
                 driver.FindElementByName("loginfmt").SendKeys(testUserName);
                 Advance();
 
+                // We need to add some delay before we enter the password to avoid
+                // clicking the button before the whole text is entered.
+                Thread.Sleep(TimeSpan.FromMilliseconds(1000));
```

Comment:
It might be a better approach to wait for the `passwd` to be available before we continue. I think you can use something like 

```suggestion
              WebDriverWait wait = new WebDriverWait(driver, 5);
              wait.until(ExpectedConditions.visibilityOfElementLocated(By.Name("passwd")));
```

Reply:
Seems like the "Commit suggestion" option automatically resolves the comment.